bundles/org.openhab.binding.echonetlite/src/main/test/org/openhab/binding/echonetlite/internal/protocol/StateCodecTest.java

```diff
@@ -0,0 +1,101 @@
+package org.openhab.binding.echonetlite.internal.protocol;
```

Comment:
In order to be discovered and run by Maven, this should be moved from:
- bundles/org.openhab.binding.echonetlite/src/main/test/org/openhab/binding/echonetlite/internal/protocol/ to:
- bundles/org.openhab.binding.echonetlite/src/test/java/org/openhab/binding/echonetlite/internal/protocol/